resources/views/myapp/add.blade.php

```diff
@@ -0,0 +1,70 @@
+<link rel="stylesheet" href="{{ asset('css/add.styles.css') }}">
+@extends('layouts.myapp')
+
+@section('title', '新規メモ')
+
+@section('content')
+    @if (count($errors) > 0)
+    <div>
+        <ul>
+            @foreach ($errors->all() as $error)
+                <li>{{ $error }}</li>
+            @endforeach
+        </ul>
+    </div>
+    @endif
+
+<form class="form" action="/myapp/add" method="post">
+<table border=1>
+    @csrf
+    <tr>
+        <td>名前：</td>
+        <td><input type="text" name="name" value="{{ old('name') }}"></td>
+    </tr>
+    <tr>
+        <td>年齢：</td>
+        <td>
+            <select name="age">
+                <option value="0">0~10歳</option>
+                <option value="1">11~20歳</option>
+                <option value="2">21~30歳</option>
+                <option value="3">31~40歳</option>
+                <option value="4">41~50歳</option>
+                <option value="5">51~60歳</option>
+                <option value="6">61~70歳</option>
+                <option value="7">71~80歳</option>
+                <option value="8">81~90歳</option>
+                <option value="9">91~100歳</option>
+            </select>
```

Comment:
データからとっこよう！
ないなら、migration seederを作って用意して、controllerでgetした変数をviewに渡して表示させよう！

Reply:
Agesテーブルを新たにDBに作成する形でやろうとしました。
create_ages_tableして、AgesTableSeederに選択肢のidと何歳～何歳ってデータをデフォでおきました。
下のエラーに遭遇して結論として解決しましたが、理由がわかりませんでした…migrateで空テーブルはできて、seedしたら下記エラー。
一旦データベース除去して新規作成、migrate, seedしたらエラー出ず。

php artisan db:seed
Seeding: Database\Seeders\MemosTableSeeder
Seeded:  Database\Seeders\MemosTableSeeder (230.90ms)
Seeding: Database\Seeders\AgesTableSeeder

   Illuminate\Database\QueryException 

  SQLSTATE[HY000]: General error: 1 no such table: ages (SQL: insert into "ages" ("agesId", "agesRange") values (0, 0~10歳))

  at C:\Users\Kohki Ota\Desktop\MyApp\vendor\laravel\framework\src\Illuminate\Database\Connection.php:678
    675▕         // message to include the bindings with SQL, which will make this exception a
    676▕         // lot more helpful to the developer instead of just the database's errors.
    677▕         catch (Exception $e) {
  ➜ 678▕             throw new QueryException(
    679▕                 $query, $this->prepareBindings($bindings), $e
    680▕             );
    681▕         }
    682▕

  1   C:\Users\Kohki Ota\Desktop\MyApp\vendor\laravel\framework\src\Illuminate\Database\Connection.php:465
      PDOException::("SQLSTATE[HY000]: General error: 1 no such table: ages")

  2   C:\Users\Kohki Ota\Desktop\MyApp\vendor\laravel\framework\src\Illuminate\Database\Connection.php:465
      PDO::prepare("insert into "ages" ("agesId", "agesRange") values (?, ?)")